index.html

```diff
@@ -107,5 +107,27 @@
   <div id="page-not-found" class="container" style="display: none">
     <%= require("html-loader!./not-found.html") %>
   </div>
+  <script>
+    if (location.toString().indexOf("darkMode") !== -1) {

```

Comment:
you can just use `location.href.includes("darkMode")`

Reply:
i can't, actually
.includes is an ES6 feature

Reply:
but now that I think about it, you're right
.href is cleaner than .toString()